src/@ionic-native/plugins/google-maps/index.ts

```diff
@@ -712,12 +712,22 @@ export class BaseClass {
  */
 export class Circle extends BaseClass {
 
-  constructor(_objectInstance: any) {
+  private _map: GoogleMap;
+
+  constructor(_map: GoogleMap, _objectInstance: any) {
     super();
+    this._map = _map;
     this._objectInstance = _objectInstance;
   }
 
   /**
+   * Return the map instance.
+   * @return {GoogleMap}
+   */
+  @CordovaInstance({ sync: true })
```

Comment:
Remove the `@CordovaIntance` decorator from the `getMap` methods.

`Cordova`, `CordovaInstance`, `CordovaProperty` and `InstanceProperty` decorators override your custom definition. Since you are defining your own functionality here, you have to remove the decorators.